### ISSUE TITLE
Message is not editing

### ISSUE THREAD

**Issue by Darker935** [Auties00/Cobalt#472]
I don't know if it's my fault or not, but I'm trying 2 ways to edited the messages and nothing is happening

The first is:
```java
var msg = whatsapp.sendMessage(message.info.parentJid(), "Just a test!").get();
out.println("Old: " + msg.toJson());

var msgEdited = new TextMessageBuilder().text("Editing message").build();

var edited = whatsapp.editMessage(msg, msgEdited).get();
out.println("Edited: " + edited.toJson());
```

And when this code, when the message have the same ID, nothing happens, but when ID is different, I receive as another message on web and unsupported on android

```java
var msg = whatsapp.sendMessage(message.info.parentJid(), "Just a test!").get();
out.println("Old: " + msg.toJson());

var msgEdited = new TextMessageBuilder().text("Editing message").build();

var key = new ChatMessageKeyBuilder()
        .id(msg.id())
        .chatJid(msg.parentJid())
        .fromMe(true)
        .senderJid(msg.senderJid())
        .build();
var info = new ChatMessageInfoBuilder()
        .status(MessageStatus.PENDING)
        .senderJid(msg.senderJid())
        .key(key)
        .message(MessageContainer.ofEditedMessage(msgEdited))
        .timestampSeconds(Clock.nowSeconds())
        .broadcast(msg.parentJid().hasServer(JidServer.BROADCAST))
        .build();
var edited = whatsapp.sendMessage(info).get();
out.println("Edited: " + edited.toJson());```

**Comment by Darker935** [comment]
Ok, I solved this issue. 

The problem is that after the update some time ago, it's nnecessary now to pass the edited message within the protocolMessage, and the lib is just passing the editedMessage, i'll create a PR soon

**Comment by Darker935** [comment]
Ok, I solved the problem.

In some past update, the necessary parameter for editing messages became protocolMessage instead editedMessage, so, the change needed on [Whatsapp.java](https://github.com/Auties00/Cobalt/blob/master/src/main/java/it/auties/whatsapp/api/Whatsapp.java) is something like that
```java
var key = new ChatMessageKeyBuilder()
        .id(msg.id())
        .chatJid(msg.parentJid())
        .fromMe(true)
        .senderJid(msg.senderJid())
        .build();

var protocol = new ProtocolMessageBuilder()
        .protocolType(ProtocolMessage.Type.MESSAGE_EDIT)
        .key(key)
        .editedMessage(new MessageContainerBuilder().textMessage(newMessage).build())
        .build();

var info = new ChatMessageInfoBuilder()
        .status(MessageStatus.PENDING)
        .senderJid(msg.senderJid())
        .key(key)
        .message(MessageContainer.of(protocol))
        .timestampSeconds(Clock.nowSeconds())
        .broadcast(msg.parentJid().hasServer(JidServer.BROADCAST))
        .build();

whatsapp.sendMessage(info);
```

**Comment by Darker935** [comment]
Solved on #473

**Comment by Auties00** [comment]
I'll review the pr now
